arches/app/media/js/viewmodels/workflow.js

```diff
@@ -301,6 +301,14 @@ define([
             });
         };
 
+        this.reverseWorkflowTransactions = function() {
+            $.ajax({
```

Comment:
Overall everything looks good, just one nit:

we're using a `GET` here to roll back the DB, yeah? Can this be a POST instead?

Reply:
Good idea - done.